src/browser_action/components/StudyInvitation.css

```diff
@@ -0,0 +1,52 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/** Components for the StudyInvitation screen */
+
+.title-bar.study {
```

Comment:
If we ever add a permanent screen with a layout this similar to the onboarding, we should factor out the styles to be reused between the two. For this temporary page, copying them here like you've done is a fine solution.